packages/create-pantheon-decoupled-kit/src/templates/next-drupal-search-api-addon/pages/search/[[...alias]].jsx

```diff
@@ -71,12 +91,34 @@ export async function getServerSideProps(context) {
 			anon: true,
 		});
 
+		const [searchTerm] = alias ? alias : [null];
+		const searchResults = searchTerm
+			? (
+					await getDrupalSearchResults({
+						apiUrl: process.env.BACKEND_URL,
+						locale: locale,
+						query: searchTerm,
+						response: res,
+					})
+			  ).data.map((value) => {
```

Comment:
This map means that the first half of this ternary will always be true, even if it is an empty array. Not sure the ternary is necessary.

Reply:
nvm I think I misread the ternary, but I still think it might be better to check for `searchResults.length` instead of sending `searchResults || null` to the client.